### PR TITLE
Add helper flow

### PR DESCRIPTION
**Pull request by Justus-Saringer**

### CODE REVIEW

File: demo/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -616,15 +616,50 @@
...
+            android:name=".features.sideeffects.ComposableLaunchedEffectActivity"
+            android:exported="false" />
+
+        <!-- Disposable Effect -->
+
+        <activity
+            android:name=".features.sideeffects.ComposableDisposableEffectActivity"
+            android:exported="false" />
+
+        <!-- Helper (Layout) -->
```
Comment (Darius3001) [inline demo/src/main/AndroidManifest.xml:631]: you forgot this comment

File: demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+                top.linkTo(parent.top)
+                end.linkTo(parent.end)
+                bottom.linkTo(flowOptions.top)
+            }
+        ) {
+
+            FlowColumn(
+                mainAxisSize = verticalFlowStateHolder.sizeMode,
+                mainAxisAlignment = verticalFlowStateHolder.mainAxisAlignment,
+                crossAxisAlignment = verticalFlowStateHolder.crossAxisAlignment,
```
Comment (Darius3001) [inline demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt:112]: have you found a reason yet why this doesnt work? i think  that there is some issue with the outer box. look comment above

File: demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+        val (flowView, flowOptions) = createRefs()
+
+        Box(modifier = Modifier
+            .constrainAs(flowView) {
+                start.linkTo(parent.start)
+                top.linkTo(parent.top)
+                end.linkTo(parent.end)
+                bottom.linkTo(flowOptions.top)
+            }
```
Comment (Darius3001) [inline demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt:106]: try adding `.fillmaxsize()` to modifier and `contentAlignment = Alignment.Center`

Comment (Justus-Saringer) [reply demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt:106]: I tried different stuff, as well your suggestion, but nothing works. I don't know why it's not working.

File: demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt (hunk 2 above)
Comment (Justus-Saringer) [reply demo/src/main/java/de/check24/compose/demo/features/constraintlayout/flow/ComposableHelperFlowVerticalActivity.kt:112]: Comment below

**Review by Darius3001 — APPROVED** [review]

**Pull request merged**